src/main/java/com/dreamhorizon/enchantments/command/EnchantCommand.java

```diff
@@ -131,4 +134,25 @@ public static void reload(CommandSender sender) {
         // loadLootTables will also reload them.
         LootTableHandler.getInstance().loadLootTables();
     }
+
+    @Subcommand("info")
+    @CommandPermission("dhenchantments.enchant.info")
+    @CommandCompletion("@dhenchantments")
+    public static void info(CommandSender sender, DHEnchantment enchantment) {
+        if (enchantment == null) {
+            // TODO: Customize message
+            sender.sendMessage("You need to specify an enchant (/dhenchant <enchant>)");
+        } else {
+            int maxLevel = enchantment.getMaxLevel();
+            String name = enchantment.getName();
+            String goesOn = enchantment.getGoesOn();
+            String description = enchantment.getDescription();
+            sender.sendMessage("===================");
+            sender.sendMessage("Name: " + name);
+            sender.sendMessage("Description: " + description);
+            sender.sendMessage("Max Level: " + maxLevel);
+            sender.sendMessage("Applies to: " + goesOn);
+            sender.sendMessage("===================");
```

Comment:
Maybe add some colour formatting to these.
for example

Navy Blue and Gold
"&9Attribute: &6"
It will look really bland without some colour to differentiate them